xarray/plot/plot.py

```diff
@@ -548,8 +548,7 @@ def _plot2d(plotfunc):
         always infer intervals, unless the mesh is irregular and plotted on
         a map projection.
     subplot_kws : dict, optional
-        Dictionary of keyword arguments for matplotlib subplots. Only applies
-        to FacetGrid plotting.
+        Dictionary of keyword arguments for matplotlib subplots.
```

Comment:
Lets also add "only used for 2D and FacetGrid plots" (if I'm, understanding this correctly)